openff/toolkit/typing/engines/smirnoff/forcefield.py

```diff
@@ -1134,6 +1138,160 @@ def parse_smirnoff_from_source(self, source):
         msg += f"Parsing failed with the following error:\n{exception_msg}\n"
         raise IOError(msg)
 
+    def _terms(self):
```

Comment:
I originally wrote all methods to start with _term, to have things organized better when trying to read docs and parse the autofill completers easier. For example, this is why I named `_terms_set_active` rather than `_set_active_terms`. I am only partially partial to the naming convention here, so if you think a `set`/`get` convention is better, I can switch to that. What do you think?

Reply:
I grant that grouping up this corner of the API in the docs/tabcompletion is a nice plus, but I think the readability of `_get_this_thing`/`_set_this_thing`is significantly more valuable. I'm not sure the right adjective to use here (maybe descriptive or expressive variable names?).

Reply:
Sure, I can make the change; and I agree that a nice clean `get`/`set` interface is a good thing. Another reason for the direction I took is that the TK API uses properties rather than set/get, so having names starting with get makes it feel like either everything or nothing should be `get`able.

Reply:
I haven't thought much about moving these getters and setters over to properties and I don't feel strongly that we need to decide that now. That could be moved in the future "private to public" switch with the prepended underscores are removed.